Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM golang:alpine AS build-env
+
+# Set up dependencies
+ENV PACKAGES git build-base
+
+# Set working directory for the build
+WORKDIR /go/src/github.com/aragon/aragon-chain
+
+# Install dependencies
+RUN apk add --update $PACKAGES
+RUN apk add linux-headers
+
+# Add source files
+COPY . .
+
+# Make the binary
+RUN make build
+
+# Final image
+FROM alpine
+
+# Install ca-certificates
+RUN apk add --update ca-certificates
+WORKDIR /root
+
+# Copy over binaries from the build-env 
+COPY --from=build-env /go/src/github.com/aragon/aragon-chain/build/aragonchaind /usr/bin/aragonchaind
+COPY --from=build-env /go/src/github.com/aragon/aragon-chain/build/aragonchaincli /usr/bin/aragonchaincli
+
+# Run ethermintd by default
+CMD ["aragonchaind"]
```

Comment:
This is only for the node but no JSON rpc right?
For our deployment we need containers exposing 2 ports for api and p2p discovery

Reply:
exposed the ports in the dockerfile. the `aragonchaincli` is the binary for the json rpc rest-server. 

I changed the entrypoint to bash shell so you can pass the command to run that binary.